resources/sdk/clisdkclient/extensions/utils/render.go

```diff
@@ -1,12 +1,30 @@
 package utils
 
 import (
+	"encoding/json"
 	"fmt"
 
+	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/data_format"
 	"github.com/tidwall/pretty"
+	"sigs.k8s.io/yaml"
 )
 
 func Render(data string) {
-	result := pretty.Pretty([]byte(data))
-	fmt.Printf("%s", result)
+	if data_format.OutputFormat == "json" && IsJSON(data) {
```

Comment:
Looks like nothing will be printed if the data isn't JSON. You could leave it printing the data and returning if the data isn't JSON.
Then you just need an if/else checking if the OutputFormat is json or yaml and render as necessary